web/src/main/java/com/google/sps/servlets/CommentsUrl.java

```diff
@@ -0,0 +1,64 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.cloud.datastore.Datastore;
+import com.google.cloud.datastore.DatastoreOptions;
+import com.google.cloud.datastore.Entity;
+import com.google.cloud.datastore.FullEntity;
+import com.google.cloud.datastore.KeyFactory;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.jsoup.select.Elements;
+
+// Get the urls of the latest deiscussion sections from reddit wallstreet bets
+@WebServlet("/store-comments-urls")
+public class CommentsUrl extends HttpServlet {
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+    Datastore datastore = DatastoreOptions.getDefaultInstance().getService();
+    KeyFactory keyFactory = datastore.newKeyFactory().setKind("Comments-url");
+
+    String Reddit =
+        "https://www.reddit.com/r/wallstreetbets/search?q=flair_name%3A%22Daily%20Discussion%22&restrict_sr=1&sort=new";
```

Comment:
Make this static member variable. Static strings don't need to be generated every time the function is called. Static variables are all caps with underscores between words.

```
@WebServlet("/store-comments-urls")
public class CommentsUrl extends HttpServlet {
  public static String REDDIT_DISCUSSION_URL = "..."

```

Reply:
How can I make it static?

![image](https://user-images.githubusercontent.com/60591404/113604941-f726eb00-9613-11eb-9219-433a528e380e.png)

Reply:
I changed it to: final String REDDIT_DISCUSSION_URL = "..."